cvat/apps/quality_control/quality_reports.py

```diff
@@ -417,20 +462,40 @@ def _fields_dict(self, *, include_properties: Optional[List[str]] = None) -> dic
             ]
         )
 
+    _CACHED_FIELDS = ["frame_count", "mean_conflict_count", "frame_share"]
+
+    def __init__(self, *args, **kwargs):
+        # these fields are optional, but can be computed on access
+        for field_name in self._CACHED_FIELDS:
+            if field_name in kwargs:
+                setattr(self, field_name, kwargs.pop(field_name))
```

Comment:
This block could probably be factored out.

Reply:
Updated